### PR TITLE
LRDOCS-6679 Access to the SLA has changed, see https://issues.liferay.com/browse/LPS-89016

### PR DESCRIPTION
**Pull request by inacionery**

### CODE REVIEW

**Comment by rbohl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: user/articles/workflow/03-workflow-metrics-reports.markdown [hunk 1]
```diff
@@ -61,7 +61,7 @@ report and you'll see a more detailed table including the following columns:
 **ID**
 : This is the workflow item's numeric identifier to the system. Importantly, you
 can click this to enter the Item Detail view. If you're doing something
-programmatically, this represents the
+programmatically, this represents the instance.
```
Comment (rbohl) [inline user/articles/workflow/03-workflow-metrics-reports.markdown]: @inacionery I think we should just delete this last sentence.

> If you're doing something programmatically, this represents the instance.

I don't think it's helpful to say this here in the User docs. Let me know if you disagree.

Comment (inacionery) [reply user/articles/workflow/03-workflow-metrics-reports.markdown]: ok, we could remove it.

**Comment by rbohl** [comment]
Processed and forwarded: https://github.com/sez11a/liferay-docs/pull/4186

**Pull request closed without merging**